README.md

```diff
@@ -11,25 +11,49 @@ npm install -g cargo-cp-artifact
 ## Usage
 
 ```
-cargo-cp-artifact [crate-name=artifact-kind=output-file] -- [wrapped-command]
+cargo-cp-artifact --artifact [artifact-kind] [crate-name] [output-file] -- [wrapped-command]
```

Comment:
Aren't these all mandatory now? The brackets convention usually means an optional argument.